.github/workflows/dependabot_update_rbis.yml

```diff
@@ -0,0 +1,71 @@
+name: Update gem RBIs on Dependabot PRs
+
+on:
+  workflow_call:
+    inputs:
+      dependency_setup_command:
+        description: A bash script that sets up manually-required dependencies
+        type: string
+        required: false
+      tapioca_env_vars:
+        description: A space-delimited list of environment variables and their values, to be passed to the `tapioca gem` command
+        type: string
+        required: false
+      valid_actors:
+        description: A stringified list of GitHub actors that can trigger this job
+        type: string
+        default: '["dependabot[bot]"]'
+        required: false
+
+permissions:
+  contents: write
+  pull-requests: write
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  update-gem-rbis:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    if: ${{ github.actor == '"dependabot[bot]"' }}
+    steps:
+      - name: Checkout PR branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+      - name: Set up any manually specified dependencies
+        if: ${{ inputs.dependency_setup_command != '' }}
+        run: ${{ inputs.dependency_setup_command }}
+        env:
+          DEBIAN_FRONTEND: noninteractive
```

Comment:
Is this necessary?

Reply:
As we don't have extra dependencies to install, we can remove this step.

Reply:
If the goal of this workflow to be picked up by others they could need this step.

Reply:
Some gems that are not released as binary but compiled require certain packages to be present. We won't run into this with our simple repos but it would be needed depending on the gem. It could be a future improvement too.